app/controllers/green_lanes/check_your_answers_controller.rb

```diff
@@ -0,0 +1,41 @@
+module GreenLanes
+  class CheckYourAnswersController < ApplicationController
+    include GreenLanesHelper
+
+    before_action :check_green_lanes_enabled,
+                  :disable_switch_service_banner,
+                  :disable_search_form
+    before_action :goods_nomenclature, only: :new
+    before_action :set_instance_variables, only: :new
+
+    def new
+      @category_one_assessments = determine_category.cat1_with_exemptions
+      @category_two_assessments = determine_category.cat2_with_exemptions
+      @ans = params[:ans]
+      @c1ex = params[:c1ex]
+      @c2ex = params[:c2ex]
+    end
+
+    private
+
+    def goods_nomenclature
+      @goods_nomenclature ||= GreenLanes::FetchGoodsNomenclature.new(results_params).call
+    end
+
+    def set_instance_variables
```

Comment:
It is probably clearer if we put all the assignments inside the `new` action.

Reply:
there was quite a lot of refactoring i was doing and yeah can change back it just started to look a bit messy in my eyes